_posts/2020-01-06-2019-reading-in-review.md

```diff
@@ -0,0 +1,501 @@
+---
+layout: post
+title: "2019 Reading in Review"
+date: "2020-01-06 12:00:00 -0600"
+---
+
+I used to review books on [a dedicated Mastodon
+account](https://wandering.shop/@benhamill), but I've since wanted to be able to
+show them to people outside the context of the fediverse. This was also spurred
+by the fact that, after moving, I read a ton of books without keeping up with
+the reviews, so I'd intended to do bunch of catch-up reviews… and that was in
+October 2018.
+
+So what's _actually_ transpired is that I've kept a one-line entry for every
+book I've read including the title, author and finish date except for the stuff
+I read right at the end of 2018 and the start of 2019. So I'm going to share
+that list here and maybe talk about some of them a bit and see how I feel.
+
+Also, I want to point out that this list doesn't include comics, which I mostly
+read in trade paperback form, or stuff I read with my kids, which is mostly very
+short but has begun to include some chapter books. Many of those things have
+been lovely and totally worth recommending or talking about, but I figure I have
+to limit scope somehow and I didn't happen to think to put those into my
+journal.
+
+
+## The Numbers
+
+Let's do some stats, shall we? Obviously I have to excluse some stuff from the
+duration stats on the basis of not knowing when I read it.
+
+**Number of Things Read:** 47
+
+**Reading Time** (including weekends, during which I generally don't read):
+* Maximum: 21 days (_Jacobin 33_ and _The Prey of Gods_)
+* [Median](https://en.wikipedia.org/wiki/Median): 8 days (_A Conjuring of Light_ and _Jacobin 34_)
+* Minimum: 0 days (_Artificial Condition_, _Exit Strategy_, _Capitalism & the
+  State_)
+* [Mode](https://en.wikipedia.org/wiki/Mode_(statistics)): 13 days (_Washington Black_, _The Body Keeps
+  the Score_, _Broken Stars_ and _The City of Brass_)
+
+
+## Things I Read in 2019
+
+The format is the title in italics, then the author's name in normal text (for
+things like books with a low number of authors) and then the completion date in
+parenthesis.
+
+
+### _After On_ by Rob Reid (???)
+
+I remember liking this book for the most part and a lot of the ideas were quite
+interesting, but it had a fondness and adulation for Silicon Valley start-up
+culture that I really couldn't stand and the resolution to the main conflict
+boiled down to what a lot of authoritarian fiction (like super hero stories)
+does: A small number of smart/good/brave/powerful/special individuals should
+stand between everyone else and the problem. Bleh.
```

Comment:
Aside: Do you have recommendations for anything where a federated/anarchic solution is plausibly implemented to handle such issues?